### PR TITLE
Static IP settings documentation

### PR DESCRIPTION
**Pull request by ptsarath**
Add Static IP settings documentation.

Signed-off-by: sarath P T <Sarath_PT@mentor.com>

### CODE REVIEW

File: docs/FAQ.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Then set up the DNS server::
+
+    $ sudo nmcli con mod 'Wired connection 1' ipv4.dns <DNS_IP>
+
+change the addressing from DHCP to static::
+
+    $ sudo nmcli con mod 'Wired connection 1' ipv4.method manual
+
+Check the changes should shown as below::
```
Comment (chombourger) [inline docs/FAQ.rst:34]: Settings for the updated configuration may be checked with `nmcli device show` as shown below::

File: docs/FAQ.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    $ nmcli device show
+    GENERAL.DEVICE:                         eth0
+    GENERAL.TYPE:                           ethernet
+    GENERAL.HWADDR:                         9E:55:11:EF:14:42
+    GENERAL.MTU:                            1500
+    GENERAL.STATE:                          100 (connected)
+    GENERAL.CONNECTION:                     Wired connection 1
+    GENERAL.CON-PATH:                       /org/freedesktop/NetworkManager/ActiveConnection/1
+    WIRED-PROPERTIES.CARRIER:               on
+    IP4.ADDRESS[1]:                         <STATIC_IP>/25
```
Comment (chombourger) [inline docs/FAQ.rst:46]: I would leave addresses you got on your end so people see a real-case setting

File: docs/FAQ.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    IP4.GATEWAY:                            <GATEWAY_IP>
```
Comment (chombourger) [inline docs/FAQ.rst:47]: same here and below

File: docs/FAQ.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    GENERAL.STATE:                          10 (unmanaged)
+    GENERAL.CONNECTION:                     --
+    GENERAL.CON-PATH:                       --
+    IP4.ADDRESS[1]:                         127.0.0.1/8
+    IP4.GATEWAY:                            --
+    IP6.ADDRESS[1]:                         ::1/128
+    IP6.GATEWAY:                            --
+    IP6.ROUTE[1]:                           dst = ::1/128, nh = ::, mt = 256
+
+Static Ip which has set should  ping to different subnet.
```
Comment (chombourger) [inline docs/FAQ.rst:69]: You may then use `ping` or other network utilities to check your connectivity.

File: docs/FAQ.rst (hunk 2 above)
Comment (ptsarath) [reply docs/FAQ.rst:46]: so can i provide a random address here ?

**Comment by chombourger** [comment]
please add `faq` to the list of documents to be included in the generated documentation - see docs/index.rst
Let's place it after `dev`

File: docs/FAQ.rst (hunk 2 above)
Comment (chombourger) [reply docs/FAQ.rst:46]: yes or the IP address you used in your testing

**Comment by chombourger** [comment]
Use `git rebase -i origin/master` to squash the second commit with the first one

**Review by chombourger — APPROVED** [review]

**Comment by chombourger** [comment]
We just need the 2 commits to be squashed and we may then merge this PR
(use git rebase -i origin/master and replace the second pick command with squash)

**Comment by chombourger** [comment]
> We just need the 2 commits to be squashed and we may then merge this PR (use git rebase -i origin/master and replace the second pick command with squash)

you are almost there. For some reason you have retained the log of the second commit
let's git commit --amend your topmost commit to read

`docs: describe how to use static IP addresses in the FAQ`

(keep the Signed-off-by line)

**Comment by ptsarath** [comment]
@chombourger  i incorporate the review comments, you can have a look now.

**Pull request merged**